common/numerated/src/interval.rs

```diff
@@ -441,7 +441,7 @@ mod tests {
     #[test]
     fn size() {
         assert_eq!(Interval::<u8>::try_from(11..111).unwrap().size(), Some(100),);
-        assert_eq!(Interval::<u8>::try_from(..1).unwrap().size(), Some(1),);
+        assert_eq!(Interval::<u8>::from(..1).size(), Some(1),);
```

Comment:
What is the difference between `try_from` + `unwrap` and `from`?

Reply:
clippy reported that this code is infallible and suggested to use `From::from`